### PR TITLE
Acom 7422 p4 p alive on ak

### PR DESCRIPTION
**Pull request by MarkWang2**
#### Description
Add researched p4p source system  to kids

### CODE REVIEW

**Comment by Roger-Tong** [comment]
Please bump up the version

**Comment by Roger-Tong** [comment]
@MarkWang2 Please check the inline comment also

File: lib/actv/asset_source_system.rb [hunk 1]
```diff
@@ -24,7 +25,7 @@ module AssetSourceSystem
     end
 
     def kids_friendly_source_system?
-      activenet? || awcamps30? || acm? || researched? || leagueone? || tennislinkteam? || tennislinktournament? || tennislinkusta? || awendurance?
+      researched? || p4p? || activenet? || awcamps30? || acm? || researched? || leagueone? || tennislinkteam? || tennislinktournament? || tennislinkusta? || awendurance?
```
[the remark this answers is not in the document]
Comment (MarkWang2) [reply lib/actv/asset_source_system.rb]: yes

[the remark this answers is not in the document]
Comment (Roger-Tong) [reply lib/actv/asset_source_system.rb]: Will you correct this so that we can publish actv sooner?

**Comment by RogerTongTong** [comment]
Looks great!

**Pull request merged**